variables.tf

```diff
@@ -340,6 +340,18 @@ variable "log_analysis_targets" {
   sensitive   = true
 }
 
+# Cloud Logs Targets
+variable "cloud_log_targets" {
```

Comment:
I suggest to rename this to `at_cloud_logs_targets` in the root level module so its clear this will create a target for atracker. And then to be consistent, rename the other ones too: `at_log_analysis_targets`, `at_eventstreams_targets`, `at_cos_targets`. No need to change them in the submodule, since the submodule is only for atracker config anyway